packages/desktop-client/src/components/filters/SavedFilters.js

```diff
@@ -4,7 +4,7 @@ import { send, sendCatch } from 'loot-core/src/platform/client/fetch';
 
 import ExpandArrow from '../../icons/v0/ExpandArrow';
 import { colors } from '../../style';
-import { MenuTooltip } from '../accounts/Account';
+import { MenuTooltip } from '../accounts/Header';
```

Comment:
Sorry, just noticed that this (`MenuTooltip`) is used outside the accounts code too. Can you move it to `common`?

Reply:
Done.